### PR TITLE
Use Jsonnet for statefulset

### PR DESCRIPTION
**Pull request by mbarbero**
Also provides Jenkins LTS 2.240.2

### CODE REVIEW

File: globals.jsonnet [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
+  docker: {
+    repository: "mbarbero",
```
Comment (fredg02) [inline globals.jsonnet]: Shouldn't that be "eclipsecbi"?

File: jenkins-master-base/releases.libsonnet [hunk 2]
```diff
@@ -1,11 +1,10 @@
...
     [r.jenkins.version]: r for r in [
-      import "2.190.1/build-args.json",
-      import "2.176.2/build-args.json",
-      import "2.176.1/build-args.json",
+      import "2.204.1/build-args.jsonnet",
+      import "2.190.1/build-args.jsonnet",
     ]
   } + {
-    latest: $.releases["2.190.1"],
+    latest: $.releases["2.204.1"],
```
Comment (fredg02) [inline jenkins-master-base/releases.libsonnet]: We should upgrade to 2.204.2 directly.

File: templates/k8s/statefulset.libsonnet [hunk 3]
```diff
@@ -23,7 +23,7 @@ local Kube = import "kube.libsonnet";
             {
               name: "jenkins",
               image: "%s:%s" % [config.docker.master.image, config.docker.master.imageTag ],
-              imagePullPolicy: "IfNotPresent",
+              imagePullPolicy: "Always",
```
Comment (fredg02) [inline templates/k8s/statefulset.libsonnet]: Is "IfNotPresent" not reliable enough?

File: globals.jsonnet (hunk 1 above)
Comment (fredg02) [reply globals.jsonnet]: Please ignore. Fixed in a later commit.

File: jenkins-master-base/releases.libsonnet (hunk 2 above)
Comment (fredg02) [reply jenkins-master-base/releases.libsonnet]: Please ignore. Fixed in a later commit.

**Comment by fredg02** [comment]
LGTM :+1:

File: templates/k8s/statefulset.libsonnet (hunk 3 above)
Comment (mbarbero) [reply templates/k8s/statefulset.libsonnet]: I don't put the sha256 anymore (makes the build process overly complicated), so we have to switch to Always to check if new version of the image is available.

**Pull request merged**